Some JNI bindings have been lost in #65

As @lorenzoPrimi noticed that #65 I clearly forgot to forward port some JNI bindings:

```
06-06 13:06:22.446 23817-23923/org.openobservatory.ooniprobe E/art: No implementation found for void io.ooni.mk.MKCollectorResubmitTask.SetSoftwareName(long, java.lang.String) (tried Java_io_ooni_mk_MKCollectorResubmitTask_SetSoftwareName and Java_io_ooni_mk_MKCollectorResubmitTask_SetSoftwareName__JLjava_lang_String_2)
06-06 13:06:22.447 23817-23923/org.openobservatory.ooniprobe E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #2
    Process: org.openobservatory.ooniprobe, PID: 23817
    java.lang.RuntimeException: An error occurred while executing doInBackground()
        at android.os.AsyncTask$3.done(AsyncTask.java:309)
        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
        at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
        at java.util.concurrent.FutureTask.run(FutureTask.java:242)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:234)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
        at java.lang.Thread.run(Thread.java:818)
     Caused by: java.lang.UnsatisfiedLinkError: No implementation found for void io.ooni.mk.MKCollectorResubmitTask.SetSoftwareName(long, java.lang.String) (tried Java_io_ooni_mk_MKCollectorResubmitTask_SetSoftwareName and Java_io_ooni_mk_MKCollectorResubmitTask_SetSoftwareName__JLjava_lang_String_2)
        at io.ooni.mk.MKCollectorResubmitTask.SetSoftwareName(Native Method)
        at io.ooni.mk.MKCollectorResubmitTask.setSoftwareName(MKCollectorResubmitTask.java:68)
        at io.ooni.mk.MKCollectorResubmitTask.<init>(MKCollectorResubmitTask.java:45)
        at org.openobservatory.ooniprobe.common.ResubmitTask.perform(ResubmitTask.java:44)
        at org.openobservatory.ooniprobe.common.ResubmitTask.doInBackground(ResubmitTask.java:103)
        at org.openobservatory.ooniprobe.common.ResubmitTask.doInBackground(ResubmitTask.java:28)
        at android.os.AsyncTask$2.call(AsyncTask.java:295)
        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:234) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588) 
        at java.lang.Thread.run(Thread.java:818) 
[snip]
```

Sorry, mate!

--- a/android-libs/src/main/cpp/collector.cpp
+++ b/android-libs/src/main/cpp/collector.cpp
@@ -43,6 +43,14 @@ MKALL_SET_STRING(MKCollectorResubmitTask_SetContent,
                  mk_collector_resubmit_request_set_content,
                  mk_collector_resubmit_request_t)
 
+MKALL_SET_STRING(MKCollectorResubmitTask_SetSoftwareName,
+                 mk_collector_resubmit_request_set_software_name,
+                 mk_collector_resubmit_request_t)
+
+MKALL_SET_STRING(MKCollectorResubmitTask_SetSoftwareVersion,
+                 mk_collector_resubmit_request_set_software_version,
+                 mk_collector_resubmit_request_t)
+
 MKALL_GET_POINTER(MKCollectorResubmitTask_Perform,
                   mk_collector_resubmit,
                   mk_collector_resubmit_request_t)

